lametro/templates/board_members/board_members.html

```diff
@@ -23,16 +23,15 @@
         <div class="col-sm-6">
             <h1 class="map-heading">
                 Metro Board of Directors
-                <a tabindex="0" class="map-info" data-container="body" data-toggle="popover" data-placement="right" title="What does this map represent?" data-content="<p style='font-family: &quot;Open Sans&quot;, sans-serif; font-size: 14px;'>The three layers of this map show areas from which members of the Metro Board of Directors are nominated:<br><br>
-                <strong>(1) Los Angeles County Districts</strong> – The elected representative from each district from the Los Angeles County Board of Supervisors serves on the Metro Board.<br><br>
-                <strong>(2) Sectors</strong> – As defined by the California League of Cities, all the incorporated and unincorporated areas in Los Angeles County, excluding the City of Los Angeles, are divided into four sectors. Each sector nominates one representative, which is approved and appointed by all members of the California League of Cities, to serve on the Metro Board.<br><br>
-                <strong>(3) City of Los Angeles</strong> – Residents within the City of Los Angeles are represented by the mayor and three city appointees on the Metro Board.<br><br>
-                Learn more about <a href='{% url 'about' %}#about-la-metro'>Metro Board appointments</a>.</p>"><h2><small> <i class="fa fa-info-circle" aria-hidden="true"></i> Learn more</small></h2><a>
             </h1>
-
         </div>
-        <div class="col-sm-6">
-            <h4>Look up a Metro Board Member</h4>
+    </div>
+</div>
```

Comment:
Any reason to have multiple container elements, or can we consolidate into one?

Reply:
We absolutely can. This was a leftover from when I moved the Learn More tooltip and I hadn't considered removing these, so that's for sure the move!